src/stan-users-guide/efficiency-tuning.qmd

```diff
@@ -1279,57 +1296,69 @@ To make the model even more efficient, a transformed data variable
 defined to be `sum(y)` could be used in the place of `sum(y)`.
 
 
-## Standardizing predictors and outputs
+## Standardizing predictors
 
-Stan programs will run faster if the input is standardized to have a
+Stan programs will run faster if the data is standardized to have a
 zero sample mean and unit sample variance.  This section illustrates
 the principle with a simple linear regression.
 
 Suppose that $y = (y_1,\dotsc,y_N)$ is a sequence of $N$ outcomes and
-$x = (x_1,\dotsc,x_N)$ a parallel sequence of $N$ predictors.  A
-simple linear regression involving an intercept coefficient $\alpha$
-and slope coefficient $\beta$ can be expressed as
+$X$ is a matrix of size $N \times J$ where $J$ is the number of predictors.
+Each column $j$ of $X$ represents a feature vector $\textbf{x}_j$.
+
+A linear regression involving an intercept coefficient $\alpha$
+and an $J$-length vector of slope coefficients $\mathcal{B}$ can be expressed as
 $$
-y_n = \alpha + \beta x_n + \epsilon_n,
+y_n = \alpha + \mathcal{B} X[n, ] + \epsilon_n,
 $$
 where
 $$
 \epsilon_n \sim \textsf{normal}(0,\sigma).
 $$
 
-If either vector $x$ or $y$ has very large or very small values or if the
-sample mean of the values is far away from 0 (on the scale of the values),
-then it can be more efficient to standardize the outputs $y_n$ and
-predictors $x_n$.  The data are first centered by subtracting the
-sample mean, and then scaled by dividing by the sample deviation.
-Thus a data point $u$ is standardized with respect to
-a vector $y$  by the function $\textsf{z}_y$, defined by
+If any of the columns of $X$ has a mean very far from $0$
+or a standard deviation far away from $1$,
+then it can be more efficient to standardize the columns of $X$
+so that each column has a mean of zero and standard deviation of 1.
+The data are first zero-centered by subtracting the
+column mean and then scaled by dividing by the column standard deviation.
+
+The mean $\mu_j$ of column $j$ is given by:
+
 $$
-\textsf{z}_y(u) = \frac{u - \bar{y}}{\texttt{sd}(y)}
+\mu_j = \frac{1}{N} \sum_{i=1}^{N} x_{ij}
```

Comment:
I've been using "x_{i, j}" with the comma because we use a bunch of multi-letter indexes where concatenation can be confusing.

Also, I tried to be consistent in the doc and sample programs that we want the sum {n=1}^N if we have N elements.  It saves alphabetic characters and makes the formulas easier to align.